pkgs/by-name/ta/talhelper/package.nix

```
@@ -0,0 +1,59 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  lib,
+  versionCheckHook,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "talhelper";
+  version = "3.0.28";
+
+  src = fetchFromGitHub {
+    owner = "budimanjojo";
+    repo = "talhelper";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-ZAzj0KMGtm0817zKur088fAoYDMcnrlDXY+DjP5PIVY=";
+  };
+
+  vendorHash = "sha256-5cuOUGoSuuc5Ro1XVQGgW+flp3rqaXNhDewzeErq4k0=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/budimanjojo/talhelper/v3/cmd.version=v${finalAttrs.version}"
+  ];
+
+  subPackages = [
+    "."
+    "./cmd"
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+
+  postInstall = ''
```

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

https://github.com/NixOS/nixpkgs/issues/308283